python/distributed-ucxx/distributed_ucxx/ucxx.py

```diff
@@ -378,8 +384,8 @@ def __init__(  # type: ignore[no-untyped-def]
         self._ep = ep
         self._ep_handle = int(self._ep._ep.handle)
         if local_addr:
-            assert local_addr.startswith("ucxx")
-        assert peer_addr.startswith("ucxx")
+            assert local_addr.startswith(("ucx://", "ucxx://"))
```

Comment:
Can this be tested yet, or not until the other PR is merged?

Reply:
Not until https://github.com/rapidsai/rapids-dask-dependency/pull/116 is merged, and I think not even after, because we can't use another protocol to hit this conditional, or at least not easily. Probably not worth the time invested at the moment.